src/content/getting-started/designers/designers.md

```diff
@@ -2,16 +2,38 @@
 title: Designers
 ---
 
-This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.
+## Resources
```

Comment:
Can we change to "Get the kit" and the two tiles are the different ways of accessing it.